src/druidry/filters.py

```diff
@@ -139,6 +139,11 @@ class SelectorFilter(Filter):
     def __init__(self, dimension=None, value=None, **kwargs):
         if not isinstance(dimension, basestring):
             raise ValueError("`dimension` is required to be a string")
+        if isinstance(value, bool):
+            value = {
+                0: 'f',
+                1: 't'
```

Comment:
Why not use `False` and `True` as the dictionary keys here?

Also, I had not idea that this type of thing would work. Is this because `hash(True) == 1` and `hash(1) == 1`, so they are treated as the same key?

Reply:
Oh yea that would work and would be less confusing. I didn't realize you could use bools as keys! (And yea they are treated as the same key.)

Reply:
I fixed this then force pushed over it to keep that commit history clean. :)